packages/dev/test/ServerTestUtils.ts

```diff
@@ -56,7 +56,8 @@ export async function assertRelayAdded (transactionHashes: PrefixedHexString[],
 
 export async function getTotalTxCosts (transactionHashes: PrefixedHexString[], gasPrice: string): Promise<BN> {
   const receipts = await resolveAllReceipts(transactionHashes)
-  return receipts.map(r => toBN(r.gasUsed).mul(toBN(gasPrice))).reduce(
+  // @ts-ignore
```

Comment:
I don't see what has to be ts-ignored here, is it needed?

Reply:
receipt interface doesn't have "effectiveGasPrice", so ts complains, at least until web3 version update I guess.

Reply:
ok ok